pkg/types/default.go

```diff
@@ -18,6 +18,8 @@ const (
 	DefaultLocalAuthCollection         string        = "usercredentials"
 	GithubState                        string        = "github"
 	JWTUserCredentialsKey              string        = "userCredentials"
+	GoogleState                        string        = "google"
+	UserInfoKey                        string        = "userInfo"
```

Comment:
Where are we using this?

Reply:
I think the userInfoKey came from rebase of older-code, this can be removed, the GoogleState constant is used to decide the `auth_type`, though this constant is duplicate of the one in the models package.

Reply:
Should I keep the GoogleState constant too, the other one is a typed string AuthType `"google"`.

```bash
└─ $ ▶ git grep '"google"'
pkg/models/user.go:54:  GoogleAuth AuthType = "google"
pkg/types/default.go:21:        GoogleState                        string        = "google"
```